game/qsp.ksy

```diff
@@ -0,0 +1,347 @@
+meta:
+  id: qsp
+  endian: le
+  application: Quest Soft Player
+  file-extension: qsp
+  encoding: "utf-16le" # in fact "ucs-2le"
+  title: "New QSP format"
```

Comment:
* Can I ask you to remove the word "format" from `title`? All .ksy are "formats", so we apply that wording automatically on format.kaitai.io generation where applicable. Having "New QSP format format" makes it akward.
* If we have certain abbrevation, it's better to use "Full Name (FN)" formula.

So, in this case, I'd propose something like "Quest Soft Player (QSP) game, new version" (ideally replacing "new version" with something more precise).